tests/markFeatureWriter_test.py

```diff
@@ -16,9 +16,10 @@ def test_add_classes(self):
         glyph.appendAnchor(glyph.anchorClass(
             anchorDict={'name': '_bottom', 'x': 100, 'y': 0}))
         lines = []
-        writer = MarkFeatureWriter(
-            ufo, anchorList=(('bottom', '_bottom'),), mkmkAnchorList=(),
-            ligaAnchorList=((('top_1', 'top_2'), '_top'),))
+        writer = MarkFeatureWriter(ufo)
+        writer.anchorList = (('bottom', '_bottom'),)
+        writer.mkmkAnchorList = ()
+        writer.ligaAnchorList = ((('top_1', 'top_2'), '_top'),)
```

Comment:
I'd do this: the `setupAnchorPairs ` method is always called by default, at the end of `MarkFeatureWriter.__init__`; clients can subclass and override that method if they wish to set those attributes differently than the built-in logic. The API for mark feature writer still remains the same as other feature writers, i.e. instantiate it with a font and call `compile` on it. What's wrong with that?